package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ckeditor/ffmvic-ckeditor5-build-classic",
```

Comment:
merging this to master `ffmvic-ckeditor5-build-classic`?

Reply:
yes